server/serverMethods.js

```diff
@@ -35,6 +37,29 @@ function htmlEscape(str) {
     .replace(/;/g, '');
 };
 
+function getUserLanguage(acceptedLanguages) {
+  let language = '';
+
+  if (acceptedLanguages && acceptedLanguages[0]) {
+    language = acceptedLanguages[0];
+
+    if (language.indexOf('-') !== -1) {
+      language = language.split('-')[0];
+    } else if (language.indexOf('_') !== -1) {
+      language = language.split('_')[0];
+    }
+  }
+
+  return language;
+}
+
+function getUserCountry(req) {
+  const ip = req.headers['x-forwarded-for'] || req.connection.remoteAddress;
+  const geo = geoip.lookup(ip) || {};
+
+  return geo.country ? geo.country.toLowerCase() : '';
```

Comment:
Maybe use lodash: `return _.get(geo, 'country', '').toLowerCase;`?

Reply:
@jeffswartz  Changed, thanks for the feedback 👍